src/fetchData.js

```diff
@@ -4,9 +4,41 @@ import { sortElements } from './sortElements.js'
  * @returns {object} cityDetails
  */
 export async function fetchCityDetails () {
-  let cityDetails = await fetch('../Assets/HTML&CSS/files/data.json')
-    .then((res) => { return res.json() })
-    .then((data) => { return data })
+  const getCityDetailsUrl = 'https://soliton.glitch.me/all-timezone-cities'
```

Comment:
This and line 40 can be like this
Also let the base url const be stored globally.
```suggestion
  const weatherApiBaseUrl = 'https://soliton.glitch.me'
  const getCityDetailsUrl = `${weatherApiBaseUrl }/all-timezone-cities`
```